rllib/env/multi_agent_episode.py

```diff
@@ -755,6 +753,108 @@ def finalize(
 
         return self
 
+    def concat_episode(self, other: "MultiAgentEpisode") -> None:
+        """Adds the given `other` MultiAgentEpisode to the right side of self.
+
+        In order for this to work, both chunks (`self` and `other`) must fit
+        together. This is checked by the IDs (must be identical), the time step counters
+        (`self.t` must be the same as `episode_chunk.t_started`), as well as the
+        observations/infos at the concatenation boundaries (`self.observations[-1]`
```

Comment:
This is only true for `SAE`s, correct? In `MAE`s we keep the observations in the `SAE`s, don't we?

Reply:
fixed